_bookdown.yml

```diff
@@ -1,5 +1,7 @@
 book_filename: "bookclub-tmwr"
-output_dir: "_book"
+repo: https://github.com/r4ds/bookclub-tmwr
+edit: "https://github.com/r4ds/bookclub-tmwr/edit/main/%s"
+output_dir: "docs"
```

Comment:
This change will break the GHA deployment. Revert it to "_book".

Reply:
Also un-commit the docs folder; _book is intentionally ignored since it builds on commit.